How to Use Blockscout With ETC (#1294)
ethereumclassic/ethereumclassic.github.io: content/blog/2023-07-19-how-to-use-blockscout-with-etc/index.md

![](6.png)

## Search ETC by Block Number

1. In the same way that you look up ETC addresses or transactions on Blockscout, you can also search individual blocks by their number. For this example, just for fun, we typed in the search field the block number 1,920,000, which is the infamous block in which the Ethereum network split from ETC in 2016. You may enter the block number you wish to search, then press enter.

![](7.png)

2. In the next screen you will see that the block also has its specific URL. In the body of the page, the top box shows the statistics of the block. These include: block height, the timestamp, how many transactions it contained, which was the miner that mined the block, block size in bytes, the block cryptographic stamp or hash, the hash of the previous block or parent hash, the mining difficulty, total difficulty of the chain up to this block, the gas used, the gas limit of this block, the nonce used to create the block hash, and the miner reward, which was 5 ETC at the time in 2016.

As you may see, because ETC is the original chain and did not do TheDAO hard fork, then this is a typical and boring block!

![](8.png)

> As a comparison, the same block on the Ethereum blockchain has different information and actually has a note saying “dao-hard-fork (Hex:0x64616f2d686172642d666f726b)”.

Reviewer: Line 115: Rewrite for readability
```suggestion
As a comparison, that same block on the Ethereum blockchain has much different information. It even contains a note reading “dao-hard-fork (Hex:0x64616f2d686172642d666f726b)”!
```

![](8-5.png)

> 3. Going back to ETC block 1,920,000, if you scroll down you will be able to see the list of all the transaction on that block. 

Reviewer: Line 119: Correct plural
```suggestion
3. Going back to ETC block 1,920,000, if you scroll down you will be able to see the list of all the transactions on that block. 
```

![](9.png)

## Search ETC by Token Symbol: HEBE

> 1. Another thing that may be done on Blockscout is to search tokens by their symbol. For example, to search the HEBE token issued by the HebeBlock team, you just need to type its symbol “HEBE” on the top right hand corner field and press enter.

Reviewer: Line 125: Correct preposition; Add hyphenation; format button text
```suggestion
1. Another thing that may be done on Blockscout is to search tokens by their symbol. For example, to search the HEBE token issued by the HebeBlock team, you just need to type its symbol “HEBE” in the top right-hand corner field and press `Enter`.
```

![](10.png)

2. In the next screen, the page will contain the top box with the main data of the ERC-20 token. This includes: the contract address, total supply, how many addresses hold the token, how many transfers have been done since its inception, how many decimals it is denominated in, and the token type (usually an ERC-20 token).

![](11.png)

## Search ETC by Token Symbol: WETC

> 1. Just as we may search dapp tokens, we can also search wrapped tokens, such as WETC, on Blockscout because they are also ERC-20 tokens. To do this just enter “WETC” on the right hand side corner field and press enter.

Reviewer: Line 135: Add hyphenation; format button text
```suggestion
1. Just as we may search dapp tokens, we can also search wrapped tokens, such as WETC, on Blockscout because they are also ERC-20 tokens. To do this just enter “WETC” on the right-hand side corner field and press `Enter`.
```
